src/CowSdk.ts

```diff
@@ -6,29 +6,37 @@ import { SupportedChainId as ChainId, SupportedChainId } from './constants/chain
 import { Context, CowContext } from './utils/context'
 import { signOrder, signOrderCancellation, UnsignedOrder } from './utils/sign'
 import { ZeroXApi } from './api/0x'
-import { MatchaOptions } from './api/0x/types'
+import { ZeroXOptions } from './api/0x/types'
 import ParaswapApi from './api/paraswap'
+import { ParaswapOptions } from './api/paraswap/types'
+import { WithEnabled } from './types'
 
 type Options = {
   loglevel?: LogLevelDesc
-  matchaOptions?: MatchaOptions
+  zeroXOptions?: Partial<ZeroXOptions & WithEnabled>
+  paraswapOptions?: Partial<ParaswapOptions> & WithEnabled
 }
 
 export class CowSdk<T extends ChainId> {
   context: Context
   cowApi: CowApi
   metadataApi: MetadataApi
   cowSubgraphApi: CowSubgraphApi
-  zeroXApi: ZeroXApi
-  paraswapApi: ParaswapApi
+  zeroXApi?: ZeroXApi
```

Comment:
PR #60 fixes this entire types debacle here with the optional APIs as well as the tests where the `?` operator is used with the apis. e.g `cowSdk.paraswapApi?.getQuote(...)`